patricia/patricia_dense_test.go

```diff
@@ -197,15 +199,18 @@ func TestTrie_DeleteLeakageDense(t *testing.T) {
 
 	oldBytes := heapAllocatedBytes()
 
-	for _, v := range data {
-		if ok := trie.Insert(Prefix(v.key), v.value); ok != v.retVal {
-			t.Errorf("Unexpected return value, expected=%v, got=%v", v.retVal, ok)
+	// repeat insertion/deletion for 10K times to catch possible memory issues
+	for i := 0; i < 10000; i++ {
+		for _, v := range data {
+			if ok := trie.Insert(Prefix(v.key), v.value); ok != v.retVal {
```

Comment:
Yeah, I guess. Depends what the memory leak is :-) I will check this out when I have more time. Expect me to get back to this aroud 2042 :D